frontend/src/utils/wildcard.js

```diff
@@ -9,27 +9,29 @@
 import startsWith from 'lodash/startsWith'
 import endsWith from 'lodash/endsWith'
 import map from 'lodash/map'
-import trim from 'lodash/trim'
 import filter from 'lodash/filter'
 import head from 'lodash/head'
+import includes from 'lodash/includes'
 
 export function wildcardObjectsFromStrings (wildcardStrings) {
   return map(wildcardStrings, item => {
     let startsWithWildcard = false
     let endsWithWildcard = false
     let customWildcard = false
-    const value = trim(item, '*')
-    let pattern = value
+    let value = item
```

Comment:
rename item to value and remove this line

Reply:
we need to keep the originalValue for line 39